composer.json

```diff
@@ -23,16 +23,17 @@
         "doctrine/doctrine-cache-bundle": "~1.1",
         "friendsofsymfony/http-cache": "~1.4",
         "jms/serializer-bundle": "^1.1",
-        "friendsofsymfony/rest-bundle": "~1.5",
+        "friendsofsymfony/rest-bundle": "^1.6",
         "guzzlehttp/guzzle": "^6.2",
         "imagine/imagine": "~0.6.1",
         "massive/search-bundle": "0.16.*",
         "sulu/document-manager": "0.9.*",
-        "sensio/framework-extra-bundle": "3.0.*",
+        "sensio/framework-extra-bundle": "^3.0",
         "stof/doctrine-extensions-bundle": "^1.2.2",
         "gedmo/doctrine-extensions": "~2.4",
-        "symfony-cmf/routing-bundle": "1.2.*",
-        "symfony/symfony": "~2.8.7, !=2.8.10",
+        "symfony-cmf/routing-bundle": "^1.2 || ^2.0",
+        "symfony/symfony": "^2.8.7 || ^3.0",
+        "symfony/security-acl": "^2.8.7 || ^3.0",
```

Comment:
Why does this have to be added? The only two classes dependent on that are `SymfonyAccessControlManager` and `SymfonyAccessControlVoter`, which both have never really be used, and are marked as deprecated and to be removed in 1.2... I would say it is safe to not add this dependency and remove these two classes.